Kernel/Ordering.hpp

```diff
@@ -95,7 +93,7 @@ class Ordering
 
   virtual void show(std::ostream& out) const = 0;
 
-  static bool isGorGEorE(Result r) { return (r == GREATER || r == GREATER_EQ || r == EQUAL); }
+  static bool isGorE(Result r) { return (r == GREATER || r == EQUAL); }
```

Comment:
I think this could be called `isGreaterOrEqual` for clarity but I'm okay with this too.

Reply:
I vote for the longer name, if that's not too much work...